wazuh/wazuh-kibana-app
# Fix Index pattern setting not used

Hi team this PR solves:
- Index pattern setting not used for choosing from existing patterns

Closes #3074

### How to Test:
To test this PR first you need to create a new index pattern in Stack Management > Index pattern (if it is not created it will be created in the health check). Then you have to add this configuration to your `wazuh.yml` `pattern: '<INDEX_PATTERN_NAME>'`. After this you should clean the cookies and the caché before entering to the app. Finally click on Wazuh and the default index pattern should be the same that you had created.
